docker/etc/templates/app.ini

```diff
@@ -1,16 +1,23 @@
 [repository]
 ROOT = /data/git/repositories
 
+[repository.local]
+LOCAL_COPY_PATH = /data/gitea/tmp/local-repo
+
 [repository.upload]
 TEMP_PATH = /data/gitea/uploads
 
 [server]
 APP_DATA_PATH = /data/gitea
+LFS_CONTENT_PATH = /data/gitea/data/lfs
```

Comment:
I don't think it is good default for LFS, I think better would be `/data/git/lfs` as repositories is stored in `/data/git/repositories` directory